store/symptom.go

```diff
@@ -111,6 +114,74 @@ func (m *mongoDB) ListOfficialSymptoms(lang string) ([]schema.Symptom, error) {
 	return symptoms, nil
 }
 
+func (m *mongoDB) ListSuggestedSymptoms(lang string) ([]schema.Symptom, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+
+	lang = strings.ReplaceAll(strings.ToLower(lang), "-", "_")
+
+	if symptoms, ok := localizedSuggestedSymptoms[lang]; ok {
+		return symptoms, nil
+	}
+
+	c := m.client.Database(m.database)
+
+	query := bson.M{"source": schema.SuggestedSymptom}
+
+	cursor, err := c.Collection(schema.SymptomCollection).Find(ctx, query, options.Find().SetSort(bson.M{"_id": 1}))
+	if err != nil {
+		return nil, err
+	}
+
+	loc := utils.NewLocalizer(lang)
+
+	symptoms := make([]schema.Symptom, 0)
+
+	for cursor.Next(ctx) {
+		var s schema.Symptom
+		if err := cursor.Decode(&s); err != nil {
+			return nil, err
+		}
+
+		if name, err := loc.Localize(&i18n.LocalizeConfig{
+			MessageID: fmt.Sprintf("symptoms.%s.name", s.ID),
+		}); err == nil {
+			s.Name = name
+		} else {
+			log.WithError(err).Warnf("can not decode name")
+		}
```

Comment:
Do we translate suggested symptoms? If not, we should remove this part.

Reply:
If we do translate those symptoms, we need to consider how to translate them in customized symptoms as well.

Reply:
We also provide translations for suggested symptoms.

Reply:
After the discussion with Casey, we decided we only translate official and suggested symptoms for now.